framework/requirements/requirements.php

```diff
@@ -73,6 +73,17 @@
         INTL_ICU_VERSION . '.'
     ),
     array(
+        'name' => 'ICU Data version',
+        'mandatory' => false,
```

Comment:
I don't think so. If you're developing English-only website you're fine w/o intl at all.